vtm/src/org/oscim/map/Viewport.java

```diff
@@ -43,8 +43,12 @@
     public static final int MIN_ZOOM_LEVEL = 2;
     public static final float MIN_TILT = 0;
 
-    /* Note: limited by numTiles in TileManager to ~80° */
-    public static final float MAX_TILT = 65;
+    /**
+     * Note: limited by
+     * - numTiles in {@link org.oscim.layers.tile.TileManager#init() TileManager}
+     * - tilt of map when cutting map on near and far plane.
+     */
+    public static final float MAX_TILT = 120;
```

Comment:
- Shouldn't change functional defaults or users' apps will start behaving differently.
Most don't need large tilt limits. There is always the `setMaxTilt` method for that.
- Also the pull request is supposed to be about Javadoc. :slightly_smiling_face:

Reply:
Sorry wasn't supposed to be commited of course.